app/assets/stylesheets/partials/_blog.scss

```diff
@@ -0,0 +1,99 @@
+.blog {
+  .blog-headline {
+    border-bottom: 1px dotted $fsek-pi-silver;
+    display: block;
+    margin: 0 0 15px;
+
+    h1,
+    h3 {
+      border-bottom: 2px solid $fsek-orange;
+      display: inline-block;
+      margin: 0 0 -2px;
+      padding-bottom: 5px;
+    }
+  }
+
+  .cover-image {
+    background-position: top center;
+    background-repeat: no-repeat;
+    box-sizing: border-box;
+    display: block;
+    margin: auto;
+    margin-bottom: 20px;
+    max-height: 400px;
+    min-height: 250px;
+    width: 100%;
+  }
+
+  .info {
+    padding: 10px 0 20px;
+
+    span {
+      border-right: 1px solid $dark-gray;
+      margin-right: 9px;
+      padding-right: 9px;
+    }
+  }
+
+  .preamble {
+    font-style: italic;
+  }
+
+  .content {
+    font-size: 20px;
```

Comment:
1) This is what I meant in the comment I made two days ago! Blog posts, pages, etc. need larger font sizes, and we should set them like this. When we are done with the other PR, we can change this to use `rem` instead :smile:

2) Again, I find the font size to large :disappointed: For laptop/desktop use, I think `17px` looks great, and for mobile it _needs_ to be even smaller (I'll send you a screenshot).